### PR TITLE
Document use cases, pipes, meta

### PR DESCRIPTION
**Pull request by subdavis**
* Adds "Use cases" section that describes how you might want to use this.
* Adds all current pipes, most with nonsense descriptions that need to be changed.  Hopefully we can get someone in vision to fill these out with better, more digestible descriptions.
* Adds descriptions of all the metadata variables currently in use
* Cool buttons

This documentation is a companion to #672, serving as the rationalization for the importance of that feature.  It is my hope that we can understand most future features as fitting into the framework of use cases described in this document.  We've been developing in pursuit of a compelling holistic rapid model generation story, but parts of that story are still missing.

Screenshot from 2021-03-29 08-27-23

### CODE REVIEW

**Comment by subdavis** [comment]
This is just docs updates. 

I don't think this is perfect, but it's an improvement.  As long as there are no issues, I'd like to merge.

File: docs/Use-Cases.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+> Scenario: You want to perform analysis (detection, tracking, measurement, etc) on object types not yet covered by the community data and pre-trained analysis pipelines available. **You want to train new models based on ground-truth annotations.**
+
+---
+
+## Iterative model refinement
+
+> Scenario: You want to start with an OK model, or a model that doesn't perfectly fit your domain or use case, and make it better.
+
+You should find out thorough experimentation whether or not annotation from scratch is faster or slower than running a sub-optimal analysis workflow and then doing manual "clean-up" on the results.  Either way, **the goal is to generate ground-truth annotations** and use them to create or improve a model.
```
Comment (BryonLewis) [inline docs/Use-Cases.md:50]: `thorough` to `through`?

File: docs/Web-Version.md [hunk 2]
```diff
@@ -27,3 +34,131 @@ A user account is required to store data and run pipelines on viame.kitware.com.
...
+* Select all items and download using the menu
+    * ![Download](images/Girder/download_selected.png)
+
+## Sharing data with teams
+
+This information will be relevant to teams where several people need to work on the same data.
+
+### Concepts
+
+By default, data updloaded to your personal user space follows these conventions.
```
Comment (BryonLewis) [inline docs/Web-Version.md:91]: `updloaded` to `uploaded`

**Review by BryonLewis — CHANGES_REQUESTED** [review]
This is extensive and really good.  Just a couple minor things.

**Comment by subdavis** [comment]
Fixed TY

**Review by BryonLewis — APPROVED** [review]

**Pull request merged**